packages/kolibri-tools/build.js

```diff
@@ -20,3 +21,19 @@ __builder.buildApiSpec();
  */
 
 __builder.exportApiSpec(path.resolve(__dirname, '../kolibri'));
+
+/*
+ * Step 4: Copy the kolibri-core module dependencies to the exported kolibri module.
+ */
+
+const kolibriCorePkg = require(path.resolve(__dirname, '../../kolibri/core/package.json'));
+
+const kolibriPackageFile = path.resolve(__dirname, '../kolibri/package.json');
+
+const kolibriPkg = require(kolibriPackageFile);
+
+kolibriPkg.dependencies = kolibriCorePkg.dependencies;
+
+fs.writeFileSync(kolibriPackageFile, JSON.stringify(kolibriPkg, undefined, 2), {
+  encoding: 'utf-8',
+});
```

Comment:
wait .. so we're overwriting the package file, and then this gets checked into git?

I think this makes it unclear when to `yarn workspace kolibri-core add ...` and when to  `yarn workspace kolibri add ...`

Reply:
The `kolibri-core` package is the only one that should be added to `kolibri` is the version generated for export to an npm module. We're overwriting the package file in `packages/kolibri` in this line, can add more comments.